srcalc/src/test/java/gov/va/med/srcalc/web/view/VariableEntryTest.java

```diff
@@ -52,4 +52,22 @@ public final void testWithRetrievedValues()
         expected.put(vars.get(5).getKey(), VariableEntry.SPECIAL_NUMERICAL);
         assertEquals(expected, entry.getDynamicValues());
     }
+    
+    @Test
+    public final void testRetrievedCardiacAge()
+    {
+    	final List<AbstractVariable> vars = SampleModels.sampleVariableList();
+    	// Replace numerical age with cardiac age (a DiscreteNumericalVariable)
+    	vars.set(1, SampleModels.cardiacAgeVariable());
```

Comment:
This is a little awkward. How about adding a `SampleModels.cardiacCabgRiskModel()`? It doesn't have to be full (maybe just include gender and cardiac age or something). I think this would make the test clearer and more realistic.